cg/services/decompression_service/decompressor.py

```diff
@@ -6,6 +6,8 @@ class Decompressor:
 
     def decompress(self, source_path: Path, destination_path: Path) -> Path:
         """Decompress a file to a directory."""
+        if not source_path.exists():
+            raise FileNotFoundError(f"Source path {source_path} does not exist.")
         with zipfile.ZipFile(source_path, "r") as zip_file:
             zip_file.extractall(destination_path)
         if not self._decompressed_path_exists(destination_path):
```

Comment:
I think we should trust `extractall` and not make this assertion ourselves. What we might need to check is that the destination path exists before running `extractall` - seems as if the extractall method will crash if it does not exist, according to chatgpt

Reply:
it cannot exist before we do the extraction, it gets created there.

Reply:
This is just a check whether the source is there. It should work like this according to the tests I set up in the decompressor.